src/custom_layers.py

```diff
@@ -0,0 +1,72 @@
+from keras.engine import Layer, InputSpec
+try:
+    from keras import initializations
+except ImportError:
+    from keras import initializers as initializations
+import keras.backend as K
+
+class Scale(Layer):
```

Comment:
This class was created to help us convert DenseNet from caffe model to Keras. I am actually not using it in this PR so I can remove it and the DenseNet model as well.